src/modules/iasql_functions@0.0.1/sql/create_fns.sql

```diff
@@ -10,9 +10,6 @@ declare
     _db_id text;
     _dblink_conn_count int;
 begin
-    if (select region from aws_account limit 1) is null then
```

Comment:
Shouldn't we keep checking if we have something in `aws_account`?

Reply:
No? We would be unable to support module management and etc for GCP/Azure/etc if we keep this. It also never used these values for any reason, so it was just unnecessary gatekeeping.

Reply:
I get your point, would need to exists a similar guard for each cloud provider and probably it does not makes sense. How it be handle the case where the `aws_account` modules is not installed or it has no records? that would break things or it is being handled in other part of the code already?

Reply:
> I get your point, would need to exists a similar guard for each cloud provider and probably it does not makes sense. How it be handle the case where the `aws_account` modules is not installed or it has no records? that would break things or it is being handled in other part of the code already?

So, the `aws_account` module *has* to be installed before you can install other aws modules since they depend on it. But on top of that, you can't successfully install any other aws module until there's a single row in the `aws_account` table with your credentials because the automatic sync-on-install for that module would fail.

It is still *possible* for someone to break it after-the-fact by deleting the `aws_account` record after installing other modules, but I don't think we should block all iasql stored proc access on that, especially if/when we add some sort of function for managing these "no cloud side" modules in a more general way.

Reply:
I added this here before the `is_ready` field existed in the metadata repository for `iasql_database`. I am okay with removing this logic because eventually we would have more clouds or providers other than AWS. Right now the `is_ready` field would not function properly though, so we need to remove it, revive it, introduce a special kind of provide module or perhaps provider credentials simple live in `iasql_metadata`? `aws_account` is not a normal module and neither would a `heroku_account`, `gcp_account`, etc? That can be a follow up once we need to add more clouds though and maybe we just remove `is_ready` from the metadata and derive it in the dashboard using graphql?

Reply:
I think deriving `is_ready` from a graphql query sounds like a plan to me. :)

Reply:
Done :)